looking at build fail
```
======================================================================
FAIL: test_combined1 (custom.ewsghana.tests.handlers.stock_on_hand.StockOnHandTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/custom/ewsghana/tests/handlers/stock_on_hand.py", line 70, in test_combined1
    self.run_script(a)
  File "/home/travis/build/dimagi/commcare-hq/custom/logistics/tests/test_script.py", line 47, in run_script
    self.assertEqual(msg.text, command['text'])
AssertionError: u'Dear super, Test RMS is experiencing the following problems: below reorder level Male Condom' != 'Dear super, Test RMS is experiencing the following problems: stockouts Lofem; below reorder level Male Condom'
----------------------------------------------------------------------
```

--- a/corehq/apps/commtrack/parsing.py
+++ b/corehq/apps/commtrack/parsing.py
@@ -123,10 +123,10 @@ def ledger_json_to_stock_report_helper(form, report_type, ledger_json):
     # helper functions
     def get_date():
         timestamp = ledger_json.get('@date') or form.received_on
-        if isinstance(timestamp, datetime.date):
+        if type(timestamp) is datetime.date:
             timestamp = datetime.datetime.combine(timestamp, datetime.time())
 
-        if not isinstance(timestamp, (datetime.datetime, datetime.date)):
+        if not isinstance(timestamp, (datetime.datetime)):
             raise InvalidDate("{} has invalid @date".format(ledger_json))
         return timestamp
 

